frontend/Components/Footer.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
```

Comment:
Unrealted to this file but:
In `.eslintrc` change `@typescript-eslint/explicit-function-return-type` to `off`. React has a lot of types that are implied by the convention and it's just a waste of everything to be explicit about them in my opinion.

Reply:
Also add this:
```
'react/prop-types': ['off'],
```

Maybe here:
![image](https://user-images.githubusercontent.com/4552000/75492171-7da3de80-59b7-11ea-9aac-600551794a87.png)

Prop types are a built-in mechanism in React for runtime checking of props which we don't need because we have compile-time type checking with TypeScript.